pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +314,36 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+// getLogWithStacktracing search for the first stack trace line and offset it by logLinesOffset
+func getLogWithStacktracing(logArray []string) string {
+	var log []string
+	var found bool
+
+	for idx := range logArray {
+		line := logArray[idx]
+		// if it already found the stack we just want to keep add the log till the end
+		if found {
+			log = append(log, line)
+			continue
+		}
+		// tries to find the stack message at the given line and add it to the log
+		if found = stackTraceRegex.MatchString(line); found {
+			log = append(log, line)
+			continue
+		}
+		// if it didn't find the line yet, keep add it as offset limiting by the arbitrary
+		// offset limit value
+		log = append(log, line)
+		if len(log) > int(logLinesOffset) {
```

Comment:
`logLinesOffset` can be defined as `int` right? And we will need no retyping.

Reply:
Yep. I think that doesn't make sense to use these consts as `int64` - I'll change it.